generator/generator_tests/altitude_test.cpp

```diff
@@ -139,12 +147,16 @@ void TestAltitudes(MwmValue const & mwmValue, string const & mwmPath,
     TEST_EQUAL(altitudes.size(), pointsCount, ());
 
     for (size_t i = 0; i < pointsCount; ++i)
-      TEST_EQUAL(altitudeGetter.GetAltitude(f.GetPoint(i)), altitudes[i], ("A wrong altitude"));
+    {
+      TAltitude const fromGetter = altitudeGetter.GetAltitude(f.GetPoint(i));
+      TAltitude const expected = (fromGetter == kInvalidAltitude ? kDefaultAltitudeMeters : fromGetter);
```

Comment:
Not good to derive expectation from AltitudesGetter. All expectations should be formed independently, by the author of the test code.